src/MoBi.Core/Services/QuantityToParameterValueMapper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility;
+
+namespace MoBi.Core.Services
+{
+   public interface IQuantityToParameterValueMapper : IMapper<IQuantity, ParameterValue>
+   {
+   }
+   
+   public class QuantityToParameterValueMapper : IQuantityToParameterValueMapper
+   {
+      private readonly IEntityPathResolver _entityPathResolver;
+
+      public QuantityToParameterValueMapper(IEntityPathResolver entityPathResolver)
+      {
+         _entityPathResolver = entityPathResolver;
+      }
+
+      public ParameterValue MapFrom(IQuantity input)
+      {
+         var parameterValue = new ParameterValue
+         {
+            Path = _entityPathResolver.ObjectPathFor(input),
```

Comment:
These are the properties I could imagine changing in the context of simulation. I don't even think you can change Dimension or DisplayUnit